packages/vckit-oa-renderers/package.json

```diff
@@ -18,15 +18,15 @@
   "scripts": {
     "build": "npm run check-types && cross-env NODE_ENV=production webpack --progress --mode production",
     "check-types": "tsc --sourceMap false --noEmit",
-    "dev": "cross-env NODE_ENV=development webpack-dev-server",
-    "example:application": "webpack-dev-server --config application/webpack.config.js",
-    "example:html": "npx http-server ./application",
+    "dev": "cross-env NODE_ENV=development NODE_OPTIONS=--openssl-legacy-provider webpack-dev-server",
+    "example:application": "NODE_OPTIONS=--openssl-legacy-provider  webpack-dev-server --config application/webpack.config.js",
```

Comment:
why openssl-legacy-provider?

Reply:
Because currently, the oa renderer is using the react version 16.12.0 and the node version v18.15.0, which is why we need to have this option to prevent the error.